### ISSUE TITLE
Decide on API documentation approach

### ISSUE THREAD

**Issue by tarurar** [LykkeCorp/guidelines#9]
Let's discuss which options do we have and choose one to be used across our APIs.

Suggestions:

- [DocFX](https://dotnet.github.io/docfx/)

- [Apiary](https://apiary.io/)

**Comment by ozhelenkov** [comment]
I still do not get what's wrong with OpenAPI and Swagger. What are the real arguments against it? @cameronfletcher 

For sure we need sort of developer portal with more detailed description, examples, client libraries on different languages. 
However Swagger does things better and you have self-documented code. It can be easily customized to cover any needs.

If your API design is good enough Open API and Swagger UI will cover mostly all the consumer needs.

**Comment by KonstantinRyazantsev** [comment]
I agree with @ozhelenkov. Swagger UI is a must have anyway, because it gives you self-discoveryable API for free. Of course it would be awesome to have documentation portal with detailed API description and examples. But we should keep in mind, that it requires significant efforts to create such a documentation and even harder to keep it up to date. Thus, better to return to this question, when more important tasks will be ready.

**Comment by cameronfletcher** [comment]
I am a fan of OpenAPI (and `swagger.json`). I _hate_ SwaggerUI. There are other tools that can use `swagger.json`.
